common/schemas/kps/kps.xsd

```diff
@@ -125,6 +125,8 @@
                   <xs:element minOccurs="0" maxOccurs="1" name="DisplayFont" type="xs:string" />
```

Comment:
Would OSKFont and DisplayFont also become `km-keyboard-fonts` type?

Reply:
Ah, I see its string vs string[]

Reply:
Yeah, not changing existing fields either.

Reply:
Which is why we have separate fields for the web fonts